app/Controllers/SettingsController.php

```
@@ -615,11 +600,8 @@ public function updateAdvancedSettings(Request $request, Response $response, mys
 
     public function regenerateSitemap(Request $request, Response $response, mysqli $db): Response
     {
-        $data = (array)$request->getParsedBody();
-        if (!Csrf::validate($data['csrf_token'] ?? null)) {
-            $_SESSION['error_message'] = 'Token CSRF non valido. Riprova.';
-            return $this->redirect($response, '/admin/settings?tab=advanced');
-        }
+        $data = (array) $request->getParsedBody();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)